.github/workflows/pypi.yml

```diff
@@ -0,0 +1,102 @@
+name: PyPI
+on:
+  push:
+    branches:
+      - master
+      - auto-release
+  pull_request:
+    branches: [master]
```

Comment:
My understanding is that this would be triggered when a PR is made from master. Is that what we want?